docs/src/CommutativeAlgebra/intro.md

```diff
@@ -24,6 +24,19 @@ In describing this functionality, we will refer to quotients of multivariate pol
     Most functions discussed in this chapter rely on Gröbner basis techniques. They either execute corresponding Singular functionality, or are written as pieces of OSCAR code which rely on Singular for Gröbner basis computations.
     The functions are  implemented for multivariate polynomial rings over fields (exact fields supported by OSCAR) and, if not indicated otherwise, for multivariate polynomial rings over the integers.
 
+!!! note
+    The workhorse behind the concept of Gröbner bases is Buchberger's algorithm
+    for computing Gröbner bases. For both the concept and the algorithm a convenient way of
+    ordering the monomials appearing in multivariate polynomials and, thus, to distinguish
+    leading terms of such polynomials is needed.  Note that the performance of the algorithm
+    and the resulting Gröbner basis depend crucially on the choice of monomial ordering.
```

Comment:
You are motivating very well, why monomial orderings make sense here without naming them (in the sense of a mathematical object) explicitly and then immedidately state that the performance depends crucially on the choice of monomial ordering. This reads like a jump in the thread, unless you know the theory very well.

I suggest to avoid this by changing the last sentence to: "The choice of a suitable monomial ordering has crucial impact on the performance of the algorithm and on the resulting Gröbner basis." (This does not change the content, but helps the reader to follow along.